journal/src/main/scala/com/evolutiongaming/kafka/journal/PartitionCache.scala

```diff
@@ -527,7 +527,7 @@ private[journal] object PartitionCache {
   /**
    * State of the non-replicated journal head comparing to a given offset.
    */
-  sealed trait Result[+F[_]]
```

Comment:
I wonder, why it was required previously...

was it only to allow `sealed trait Now extends Result[Nothing]`?

Reply:
yes, exactly